public/js/components/Frames.js

```diff
@@ -8,22 +9,24 @@ const actions = require("../actions");
 const { endTruncateStr } = require("../utils/utils");
 const { getFilename } = require("../utils/source");
 const { getFrames, getSelectedFrame, getSource } = require("../selectors");
+const { List } = require("immutable");
+const classNames = require("classnames");
 
 if (typeof window == "object") {
   require("./Frames.css");
 }
 
 const NUM_FRAMES_SHOWN = 7;
 
-function renderFrameTitle(frame) {
-  return div({ className: "title" }, endTruncateStr(frame.displayName, 40));
+function renderFrameTitle({ displayName }: Object) {
```

Comment:
hmm, @Zacqary I think this style might be a bit cleaner

```js
function renderFrameTitle({ displayName: string }) {}
function renderFrameLocation({ source: Source, location: Location }) {}
```

with these imports:

```js
import type { Source, Location } from "../types";
```